images/gce-dind/Dockerfile

```diff
@@ -0,0 +1,38 @@
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM gcr.io/k8s-testimages/kubekins-e2e:v20180102-0e2b24a0b-master
+LABEL maintainer "leblancd@cisco.com"
+
+# add env we can debug with the image name:tag
+ARG IMAGE_ARG
+ENV IMAGE=${IMAGE_ARG}
+
+RUN apt-get update && apt-get install -y \
+    git \
+    wget \
+    liblz4-1 && \
+    apt-get clean
+
+# Install kubeadm-dind-cluster scripts
+RUN cd /root && \
```

Comment:
Rather than baking it in, we should pull git repos at point of use, so it's easier to change versions.

Reply:
we can do this with a second --repo bootstrap flag on the job instead of baking it into an image

Reply:
Good idea, thanks. Will change this.

Reply:
Done. Added a --repo flag for the kubeadm-dind-cluster repo.